src/lib/crypto/hash.cpp

```diff
@@ -52,26 +23,6 @@
  * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
  * POSSIBILITY OF SUCH DAMAGE.
  */
-/*
- * Copyright (c) 2005-2008 Nominet UK (www.nic.uk)
```

Comment:
I'm bit concerned about removing these copyright notices, I think it is still can be considered as a derived work. Better to keep them.

Reply:
@antonsviridenko we are slowly working through legacy inherited code and will remove inappropriate notices when those sections are fully revamped, guided by automatic code analysis.